Doc/tools/templates/layout.html

```diff
@@ -8,6 +8,19 @@
     <a href="/3/{{ pagename }}{{ file_suffix }}">{% trans %} Python documentation for the current stable release{% endtrans %}</a>.
 </div>
 {%- endif %}
+
+{%- if is_deployment_preview %}
+<div id="deployment-preview-warning" style="padding: .5em; text-align: center; background-color: #fff2ba; color: #6a580e;">
+  <div style="float: right; margin-top: -10px; margin-left: 10px;">
+    <a href="https://www.netlify.com">
+      <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" />
+    </a>
+  </div>
+  {% trans %}This is a deployment preview that was created from a <a href="{{ repository_url }}/pull/{{ pr_id }}">pull request</a>.
+  For authoritative documentation, see the {% endtrans %}
+  <a href="/3/{{ pagename }}{{ file_suffix }}">{% trans %} Python documentation for the current stable release{% endtrans %}</a>.
```

Comment:
Your [demo](https://62af84a185fcf70008d07339--steady-heliotrope-731cac.netlify.app/) links to [here](https://62af84a185fcf70008d07339--steady-heliotrope-731cac.netlify.app/3/index.html) which is a 404.

I guess the merged one will also have Netlify URL. Should this URL be an absolute link to `https://docs.python.org/3/` ?

Reply:
Good catch -- just updated